docs/Aurora/Spark job submit tool usage on aurora.md

```diff
@@ -0,0 +1,271 @@
+
+The following is a quick start for SJST with an example about how to run K-Means program with GPU, and a detailed configuration description.
+
+- [Quick Start](#quick-start)
+  - [Introduction To SJST](#introduction-to-sjst)
+  - [Create Spark Configuration Files](#create-spark-configuration-files)
+  - [Submit Spark Job](#submit-spark-job)
+  - [Submit Spark Job (interactive model)](#submit-spark-job-interactive-model)
+  - [Check Your Job Status](#check-your-job-status)
+  - [Check Output of Your Job](#check-output-of-your-job)
+  - [Command Line Help](#command-line-help)
+    - [Bash Mode](#bash-mode)
+    - [Interactive Mode](#interactive-mode)
+- [Configurations](#configurations)
+  - [env_aurora.sh](#env_aurorash)
+  - [env_local.sh](#env_localsh)
+- [Example Configuration](#example-configuration)
+
+## Quick Start
+### Introduction To SJST
+SJST is available at /lus/flare/projects/Aurora_deployment/spark/spark-job. The component of SJST is listed in the table below.
+```text
+spark-job
+├── bin/                            // The scripts that setup spark cluster and submit jobs
+├── conf-to-your-submit-dir/        // Configurations about workers and executors
+│                                   // You'll mainly work with this directory.
+├── example/                        // Example for testing and reference
+├── jars/                           // Files for loading OAP MLlib
+├── LICENSE
+├── README.docx
+└── README.txt
+
+```
+
+The following sections give a detailed example about how to submit a job.
+
+### Create Spark Configuration Files
+The spark-job/conf-to-your-submit-dir/ directory contains configuration files you want. You need to make a copy in your working directory.
+
+For example:
+```shell
+$ mkdir ~/spark_work_home
+$ cp /lus/flare/projects/Aurora_deployment/spark/spark-job/conf-to-your-submit-dir/* ~/spark_work_home
+```
+
+### Submit Spark Job
+After setup configurations, you can use the scripts in spark-job/bin/ to submit jobs from your working directory. Please make sure your configuration files are placed properly.
+
+Here are the steps to submit the dense K-Means job with 2 nodes to queue “lustre_scaling” from Aurora login node. The data path given below is a 192G CSV file from DAOS
+
+```shell
+$ cd ~/spark_work_home
+$ /lus/flare/projects/Aurora_deployment/spark/spark-job/bin/submit-spark.sh -A Aurora_deployment -l walltime=2:00:00 -l select=2 -l daos=default -q lustre_scaling /lus/flare/projects/Aurora_deployment/spark/spark-job/example/kmeans-pyspark.py daos://Intel2/hadoop_fs/HiBench/Kmeans/Input/36000000
```

Comment:
Can we have two examples here, one is reading from DAOS and another is reading from lustre or a shared file system? The tool should support both cases.